libs/ob-sql-parser/src/main/java/com/oceanbase/tools/sqlparser/statement/droptable/TableList.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.tools.sqlparser.statement.droptable;
+
+import java.util.List;
+
+import org.antlr.v4.runtime.ParserRuleContext;
+
+import com.oceanbase.tools.sqlparser.statement.BaseStatement;
+import com.oceanbase.tools.sqlparser.statement.common.RelationFactor;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+/**
+ * @Author: Lebie
+ * @Date: 2024/8/21 17:30
+ * @Description: []
+ */
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class TableList extends BaseStatement {
```

Comment:
this statement is not necessary, we may use List<RelationFactor> instead~

Reply:
I tent to keep `TableList` because it's actually a node in the grammar. Additionally, this node could help me easily get the start and stop position of all the relation factors which are more convenient for me to replace the relation factors.